tests/data/sample_specification.yaml

```diff
@@ -8,36 +8,37 @@ aea_version: '>=1.0.0, <2.0.0'
 description: 'A protocol for testing purposes.'
 speech_acts:
   performative_ct:
-    content_ct: ct:DataModel
+    content_ct: ct:DataModel7
   performative_pt:
     content_bytes: pt:bytes
     content_int: pt:int
     content_float: pt:float
     content_bool: pt:bool
     content_str: pt:str
   performative_pct:
-#    content_set_ct: pt:set[ct:DataModel] # custom type inside of set, list, and dict isn't allowed.
     content_set_bytes: pt:set[pt:bytes]
     content_set_int: pt:set[pt:int]
     content_set_float: pt:set[pt:float]
     content_set_bool: pt:set[pt:bool]
     content_set_str: pt:set[pt:str]
-#    content_list_ct: pt:list[ct:DataModel] # custom type inside of set, list, and dict isn't allowed.
+    # content_set_ct: pt:set[ct:DataModel] # custom type inside of set isn't allowed.
+    content_list_ct: pt:list[ct:DataModel6]
+    content_list_ct2: pt:list[ct:DataModel3]
```

Comment:
This is just like the line above it. Only 1 of these is enough.

Reply:
done